pkg/restapi/wallet/chapibridge/operations.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package chapibridge provides wallet server REST features related to credential handler APIs.
+package chapibridge
+
+import (
+	"net/http"
+
+	"github.com/trustbloc/edge-agent/pkg/restapi/common"
+
+	"github.com/hyperledger/aries-framework-go/pkg/controller/rest"
+)
+
+// constants for endpoints of wallet server chapi bridge controller.
+const (
+	CreateInvitationPath   = "/create-invitation"
+	RequestCHAPIAppProfile = "/request-app-profile"
+	SendCHAPIRequest       = "/send-chapi-request"
+)
+
+// Operation is REST service operation controller for CHAPI bridge features.
+type Operation struct {
+	handlers []rest.Handler
+}
+
+// Provider describes dependencies for this command.
+type Provider interface {
+}
+
+// New returns new CHAPI bridge rest controller instance.
```

Comment:
Small note: You use upper case REST in your other comments - I suggest picking one style and sticking to it (preferably upper case)